src/Chapter2.hs

```diff
@@ -740,9 +770,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains one two = [ xs | (n,xs) <- zip test1 two, test2 n]
+  where test1 = map (one `elem`) two
+        test2 = (\x -> if (x == True) then True else False)
```

Comment:
I feel like this solution is a bit overcomplicated :thinking: I see you fancy `zip` and list comprehension, but a simple combination of `filter` and partially applied `elem` function will do just as good :+1: